CHANGELOG.adoc

```diff
@@ -18,6 +18,8 @@ toc::[]
 
 === Changed
 
+* BREAKING module name changed from `hc-fhir-sdk-java` to `fhir-java`
```

Comment:
Is that related to this patch? I currently struggling to figure out, where this is happening.

Reply:
the old publishing config had an error using the project name instead of the module name, which resulted in a dependency name `care.data4life.hc-fhir-sdk-java:hc-fhir-sdk-java` and I fixed it in the new config to also use the same pattern as in all other SDKs

Reply:
👍